addons/sale/models/sale_order.py

```diff
@@ -906,6 +938,18 @@ def _action_confirm(self):
                     order._create_analytic_account()
 
     def _send_order_confirmation_mail(self):
+        """
+        Send a mail to the customer to notify him that its order has been confirmend.
+
+        :return: None
+        """
+        self._send_payment_succeeded_for_order_mail()
```

Comment:
Could you separate the two methods to not make one call the other anymore?

Reply:
I was doing that at the beginning but you will have the same code on both, the only difference is the template that is called. I would prefer if we do not have two methods that do the same stuff.

Reply:
It won't be the same code, one will have the confirmation template, and not the other. I don't see the problem here